polkadot/api/src/lib.rs

```diff
@@ -34,8 +34,10 @@ extern crate error_chain;
 #[cfg(test)]
 extern crate substrate_keyring as keyring;
 
-use client::backend::Backend;
-use client::Client;
+pub mod light;
```

Comment:
nit: since light clients are meant to be first-class we should also put the full client implementation in a submodule

Reply:
thanks, will do

Reply:
`impl<B: LocalBackend> PolkadotApi for Client<B, LocalCallExecutor<B, NativeExecutor<LocalDispatch>>>` looks like it could go in a `full` module.

Reply:
right. missed that you're talking about Api, not about the client